php/classes/handlers/class-castos-handler.php

```diff
@@ -308,7 +318,7 @@ public function upload_podcast_to_podmotor( $post ) {
 
 		$this->logger->log( 'API URL', $api_url );
 
-		$featured_image_url = $this->get_featured_image( $post );
+		$featured_image_url = $this->get_podcast_image_url( $post );
```

Comment:
As per the comment on the method below, please update this variable to $cover_image_url